pkg/kubectl/cmd/describe.go

```diff
@@ -69,10 +70,38 @@ var (
 		kubectl describe pods frontend`))
 )
 
-func NewCmdDescribe(f cmdutil.Factory, out, cmdErr io.Writer) *cobra.Command {
-	options := &resource.FilenameOptions{}
-	describerSettings := &printers.DescriberSettings{
-		ShowEvents: true,
+type DescribeOptions struct {
```

Comment:
Can this change be a separate PR, if not at least a separate commit for ease of review.

Reply:
Done